src/plays/web3-staking-dapp/smart-contract/cache/solidity-files-cache.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Can we not include the cache files?

Reply:
can you tell me the what to write in the gitignore in order to remove the cache file?
I have tried with ./smart-contract/cache/solidity-files-cache.json

Reply:
Can you try `cache/*`?

Reply:
it didn't worked

Reply:
@nirban256 , if you have already pushed the changes and they trying to ignore that files in gitignore then it won't work.

Reply:
@Sachin-chaurasiya I made another commit after editing the readme file and then it failed the test

Reply:
I don't know the exact reason for it

Reply:
I've re-run the failed test. It should pass now. Can you please take care of the comments making sure they are done?

Reply:
@atapas I could not add the cache folder in the gitignore properly this is the only issue that is left. Can you tell me how to add the folder so that git does not track it?

Reply:
ok